Samples/UIWidgetsSamples_2019_4/Assets/UIWidgetsGallery/GalleryMain.cs

```diff
@@ -19,6 +19,7 @@ protected override void main()
 
         protected override void onEnable()
         {
+            AndroidPlatformUtil.SetStatusBarValue(true);
```

Comment:
Since this should be a global setting for an app (instead of a panel), this configuration should not be set here.

Instead, we should add a static class e.g., 
static class UIWidgetsAndroidConfiguration {
    //making it true only if you want to show the status bar
    const bool showStatusBar = false;
   ...
}
where the developer can change showStatusBar to whatever they want for the app. Then we will call AndroidPlatformUtil.SetStatusBarValue once with this default value when the App starts. After that, developers can also call AndroidPlatformUtil.SetStatusBarValue whenever they want

Another thought, we should tell the user (1) if they want to make an normal app, they should change showStatusBar to true and disable the full-screen setting in player settings; (2) if they want to make a game which use UIwidgets for ui panels, they should keep showStatusBar to false and enable full-screen setting.